xdmod/entrypoint.sh

```diff
@@ -26,29 +29,43 @@ then
 
     tables=$(mysql -u${user} -p${pass} --host ${host} -NB modw -e "SHOW TABLES")
     if [[ -n "$tables" ]]; then
-        echo "XDMoD already initialized"
+        echo "Open XDMoD already initialized"
     else
-        echo "Running XDMoD setup and initial ingestion"
         #------------------------
         # Run xdmod-setup
         #------------------------
-        echo "---> Setup XDMoD SSO..."
+        echo "---> Open XDMoD Setup: SSO..."
         /srv/xdmod/scripts/xdmod-setup-sso.sh
-
+        echo "---> Open XDMoD Setup: start"
         expect /srv/xdmod/scripts/xdmod-setup-start.tcl | col -b
+        echo "---> Open XDMoD Setup: hpc resource"
         expect /srv/xdmod/scripts/xdmod-setup-jobs.tcl | col -b
+        echo "---> Open XDMoD Setup: finish"
         expect /srv/xdmod/scripts/xdmod-setup-finish.tcl | col -b
 
+        echo "Open XDMoD Import: Hierarchy"
         xdmod-import-csv -t hierarchy -i /srv/xdmod/hierarchy.csv
 
         #------------------------
         # Ingest slurm job data
         #------------------------
-        xdmod-slurm-helper -v -r hpc
-        xdmod-ingestor -v
+        echo "---> Open XDMoD Import: slurm hpc"
+        xdmod-slurm-helper -r hpc
+        echo "---> Open XDMoD: Ingest"
+        xdmod-ingestor
+
+        echo "---> Open XDMoD Setup: Job Performance"
+        expect /srv/xdmod/scripts/xdmod-setup-supremm.tcl | col -b
+        echo "---> Open XDMoD Aggregate: Job Performance"
+        aggregate_supremm.sh
```

Comment:
Do we need a check here to ensure mongodb is up and started before trying to run the aggregation? or will if fail gracefully and just work when we run it again?

Reply:
It will fail if mongodb is not running.  I am not positive how it will fail

Rerunning it again later will pick things up

Reply:
> It will fail if mongodb is not running. I am not positive how it will fail

If it exits with anything other than 0 the container won't start due to the `set -e`.

Reply:
I will look into how to do something like you do with slurm as a future PR

Reply:
> I will look into how to do something like you do with slurm as a future PR

Sounds good. Even just modifying `aggregate_supremm.sh` to just always return 0. The checking if mongodb is up code can get ugly :)